src/main/java/com/team/piickle/auth/jwt/TokenProvider.java

```diff
@@ -91,4 +91,12 @@ public String getUserId() {
         }
         return authentication.getName();
     }
+
+    public String getUserIdNullable() {
+        final Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        if (authentication == null || authentication.getName() == null) {
+            return null;
+        }
+        return authentication.getName();
```

Comment:
옵셔널 써서 메서드 명을 이렇게 바꾸는 건 어땡

```suggestion
    public Optional<String> getUserId() {
        final Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
        return Optional.nullable(authentication).flatMap(Authentication::getName);
    }
```

Reply:
오 👍 이거 작성하면서 이게 맞나 싶었는데 의견 땡큐!

Reply:
근데 이 부분이 getUserId 를 그대로 쓰면 jwt 쓰는 api랑 겹쳐서 에러나는데 결국 메소드명을 바꾸긴 해야 할 듯..?